src/pages/QuestionListsPage/QuestionLists.tsx

```diff
@@ -67,8 +66,35 @@ export default function QuestionLists(): JSX.Element {
     setModalVisibility(false);
   };
 
-  const tags = (interviewQuestions: QuestionModel[]): JSX.Element[] => {
-    return getDistinctCategories(interviewQuestions).map((category) => (
+  const showSuccessModal = (): void => {
+    const successModal = Modal.success({
+      title: `Question Set was deleted.`,
+    });
+    setTimeout(() => successModal.destroy(), 3000);
+  };
+
+  const deleteList = (id: number): void => {
+    client.deleteQuestionSet(id).then(() => {
+      setLists(lists?.filter((item) => item.questionSet?.id !== id));
+      setTimeout(() => showSuccessModal(), 500);
+    });
+  };
+
+  const showConfirmationModal = (title: string, onOk: () => any): void => {
```

Comment:
This contradicts the [no-explicit-any](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-explicit-any.md) rule, assuming that the function is not supposed to return anything this is a viable replacement

```suggestion
  const showConfirmationModal = (title: string, onOk: () => void): void => {
```